# joint_limits.yaml allows the dynamics properties specified in the URDF to be overwritten or augmented as needed

# For beginners, we downscale velocity and acceleration limits.
# You can always specify higher scaling factors (<= 1.0) in your motion requests.  # Increase the values below to 1.0 to always move at maximum speed.
default_velocity_scaling_factor: 0.5
default_acceleration_scaling_factor: 0.5

# Specific joint properties can be changed with the keys [max_position, min_position, max_velocity, max_acceleration]
# Joint limits can be turned off with [has_velocity_limits, has_acceleration_limits]
joint_limits:
  arm_elbow_joint:
    has_velocity_limits: true
    max_velocity: 3.14
    has_acceleration_limits: false
    max_acceleration: 0
  arm_shoulder_lift_joint:
    has_velocity_limits: true
    max_velocity: 3.14
    has_acceleration_limits: false
    max_acceleration: 0
  arm_shoulder_pan_joint:
    has_velocity_limits: true
    max_velocity: 3.14
    has_acceleration_limits: false
    max_acceleration: 0
  arm_wrist_1_joint:
    has_velocity_limits: true
    max_velocity: 6.28
    has_acceleration_limits: false
    max_acceleration: 0
  arm_wrist_2_joint:
    has_velocity_limits: true
    max_velocity: 6.28
    has_acceleration_limits: false
    max_acceleration: 0
  arm_wrist_3_joint:
    has_velocity_limits: true
    max_velocity: 6.28
    has_acceleration_limits: false
    max_acceleration: 0
  lh_FFJ1:
    has_velocity_limits: true
    max_velocity: 2
    has_acceleration_limits: false
    max_acceleration: 0
  lh_FFJ2:
    has_velocity_limits: true
    max_velocity: 2
    has_acceleration_limits: false
    max_acceleration: 0
  lh_FFJ3:
    has_velocity_limits: true
    max_velocity: 2
    has_acceleration_limits: false
    max_acceleration: 0
  lh_FFJ4:
    has_velocity_limits: true
    max_velocity: 2
    has_acceleration_limits: false
    max_acceleration: 0
  lh_LFJ1:
    has_velocity_limits: true
    max_velocity: 2
    has_acceleration_limits: false
    max_acceleration: 0
  lh_LFJ2:
    has_velocity_limits: true
    max_velocity: 2
    has_acceleration_limits: false
    max_acceleration: 0
  lh_LFJ3:
    has_velocity_limits: true
    max_velocity: 2
    has_acceleration_limits: false
    max_acceleration: 0
  lh_LFJ4:
    has_velocity_limits: true
    max_velocity: 2
    has_acceleration_limits: false
    max_acceleration: 0
  lh_LFJ5:
    has_velocity_limits: true
    max_velocity: 2
    has_acceleration_limits: false
    max_acceleration: 0
  lh_MFJ1:
    has_velocity_limits: true
    max_velocity: 2
    has_acceleration_limits: false
    max_acceleration: 0
  lh_MFJ2:
    has_velocity_limits: true
    max_velocity: 2
    has_acceleration_limits: false
    max_acceleration: 0
  lh_MFJ3:
    has_velocity_limits: true
    max_velocity: 2
    has_acceleration_limits: false
    max_acceleration: 0
  lh_MFJ4:
    has_velocity_limits: true
    max_velocity: 2
    has_acceleration_limits: false
    max_acceleration: 0
  lh_RFJ1:
    has_velocity_limits: true
    max_velocity: 2
    has_acceleration_limits: false
    max_acceleration: 0
  lh_RFJ2:
    has_velocity_limits: true
    max_velocity: 2
    has_acceleration_limits: false
    max_acceleration: 0
  lh_RFJ3:
    has_velocity_limits: true
    max_velocity: 2
    has_acceleration_limits: false
    max_acceleration: 0
  lh_RFJ4:
    has_velocity_limits: true
    max_velocity: 2
    has_acceleration_limits: false
    max_acceleration: 0
  lh_THJ1:
    has_velocity_limits: true
    max_velocity: 4
    has_acceleration_limits: false
    max_acceleration: 0
  lh_THJ2:
    has_velocity_limits: true
    max_velocity: 2
    has_acceleration_limits: false
    max_acceleration: 0
  lh_THJ3:
    has_velocity_limits: true
    max_velocity: 4
    has_acceleration_limits: false
    max_acceleration: 0
  lh_THJ4:
    has_velocity_limits: true
    max_velocity: 4
    has_acceleration_limits: false
    max_acceleration: 0
  lh_THJ5:
    has_velocity_limits: true
    max_velocity: 4
    has_acceleration_limits: false
    max_acceleration: 0
  lh_WRJ1:
    has_velocity_limits: true
    max_velocity: 2
    has_acceleration_limits: false
    max_acceleration: 0
  lh_WRJ2:
    has_velocity_limits: true
    max_velocity: 2
    has_acceleration_limits: false
    max_acceleration: 0
